Horizon/Horizon/Sources/Features/Notebook/Common/Data/CourseNotebookNote.swift

```diff
@@ -16,28 +16,31 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 //
 
+import Core
 import Foundation
 
 /// This is an API agnostic entity model.
 /// It's used in the interactors and can be used in the views, but will normally be translated to a view model before being used in the views.
-struct CourseNotebookNote {
-    // MARK: - Required
+extension API {
+    struct CourseNotebookNote {
```

Comment:
What I meant in my earlier review was simply an `API` prefix like `APISubmission`, `APICourse`.